selfdrive/car/volkswagen/carstate.py

```diff
@@ -267,3 +256,23 @@ def get_cam_can_parser(CP):
     ]
 
     return CANParser(DBC[CP.carFingerprint]['pt'], signals, checks, CANBUS.cam)
+
+class MqbExtraSignals:
+  # Additional signal and message lists for optional or bus-portable controllers
+  fwd_radar = ([
+    ("ACC_Wunschgeschw", "ACC_02", 0),              # ACC set speed
```

Comment:
the tuple is a little confusing. let's call them `xxx_signals` and `xxx_checks` or just put them back in the function broken out from the main lists

Reply:
I took them out of the regular function because they can appear on either of two (and in the future, three) CAN buses, and I don't want to carry and maintain multiple copies of the relevant signal and check lists. I'm onboard with clarifying how they're referenced though.